components/table/index.css

```diff
@@ -12,511 +12,348 @@
 
-		&:focus:hover {
-			--spectrum-table-icon-color: var(--highcontrast-table-icon-color-focus, var(--mod-table-icon-color-focus-hover, var(--spectrum-table-icon-color-focus-hover)));
-		}
+		.spectrum-Table-checkbox {
+			--mod-checkbox-margin-block: 0px;
 
-		&:focus-visible,
-		&.is-keyboardFocused {
-			--spectrum-table-icon-color: var(--highcontrast-table-icon-color-focus, var(--mod-table-icon-color-key-focus, var(--spectrum-table-icon-color-key-focus)));
+			min-block-size: initial;
 		}
 	}
+}
 
-	&.is-sorted-asc,
-	&.is-sorted-desc {
-		.spectrum-Table-sortedIcon {
-			display: inline-block;
-		}
+/* The quiet table's header row cells need a border-block-end to match the body cells. */
+.spectrum-Table--quiet {
+	.spectrum-Table-headCell {
+		border-block-end: var(--mod-table-border-width, var(--spectrum-table-border-width)) solid var(--highcontrast-table-divider-color, var(--mod-table-divider-color, var(--spectrum-table-divider-color)));
 	}
+}
 
-	&.is-sorted-asc {
-		.spectrum-Table-sortedIcon {
-			transform: rotateZ(-90deg);
-		}
+.spectrum-Table-sortIcon {
+	&:last-child .spectrum-Table-headCell {
+		border-start-end-radius: var(--mod-table-border-radius, var(--spectrum-table-border-radius));
 	}
 }
 
+/* Head cell column text */
 .spectrum-Table-columnTitle {
 	display: inline-block;
 }
 
-.spectrum-Table-cell--alignCenter {
-	text-align: center;
+/********* ICONS- SORT, DISCLOSURE/MENU *********/
+.spectrum-Table-sortIcon {
+	vertical-align: var(--mod-table-header-vertical-align, var(--spectrum-table-default-vertical-align));
+	margin-inline-start: var(--mod-table-sort-icon-inline-start-spacing, 0);
+	margin-inline-end: var(--mod-table-sort-icon-inline-end-spacing, var(--mod-table-visual-to-text, var(--spectrum-table-visual-to-text)));
+	transition: transform var(--highcontrast-table-transition-duration, var(--mod-table-transition-duration, var(--spectrum-table-transition-duration))) ease-in-out;
+	color: var(--spectrum-table-icon-color);
+	display: inline-block;
+}
+
+.spectrum-Table-menuIcon {
+	vertical-align: middle;
+	margin-inline-start: var(--mod-table-visual-to-text, var(--spectrum-table-visual-to-text));
+}
+
+/********* TEXT ALIGNMENT *********/
+.spectrum-Table-cell--alignStart {
+	text-align: start;
 }
 
-.spectrum-Table-cell--alignRight {
+.spectrum-Table-cell--alignEnd {
 	text-align: end;
 }
 
+/********* TABLE BODY *********/
 .spectrum-Table-body {
 	position: relative;
 	border: none;
-	border-radius: var(--mod-table-border-radius, var(--spectrum-table-border-radius));
+	border-end-start-radius: calc(var(--mod-table-border-radius, var(--spectrum-table-border-radius)) - var(--mod-table-border-width, var(--spectrum-table-border-width)));
+	border-end-end-radius: calc(var(--mod-table-border-radius, var(--spectrum-table-border-radius)) - var(--mod-table-border-width, var(--spectrum-table-border-width)));
 	display: table-row-group;
 
 	&.is-drop-target {
```

Comment:
I'm curious if anybody has ideas on the body drop zone. Right now, I get little cell borders that interrupt the body outline.

<img width="257" alt="Screenshot 2025-05-22 at 2 15 21 PM" src="https://github.com/user-attachments/assets/dfad40c4-d687-480e-a173-2ba7d30f24fa" />

Reply:
If you apply it as an `::after` for `tbody`, that will allow it to appear over the row borders. You'll also need to set the border-radius to match the end, and use `pointer-events: none`. There's possibly other side effects that I'm not seeing checking it via devtools.